<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_push_button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_push_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>UPDATE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_push_button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>DELETE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_push_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>LOAD</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="display_table_widget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>371</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>NAME : </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name_line_edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="email_label">
       <property name="text">
        <string>EMAIL : </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="email_line_edit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="contactlabel">
       <property name="text">
        <string>CONTACT : </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="contact_line_edit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
